roles/ealapps/molecule/default/verify.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: check php curl status
+    ansible.builtin.package:
+      name: "{{ item }}"
+      state: present
+    check_mode: true
+    register: pkg_status
+    loop:
+      - php8.1-curl
```

Comment:
Why are we just checking for php8.1-curl, and not the other php8.1 packages (like php8.1, php8.1-dev, php8.1-zip) here?

Reply:
this come from the PHP role. https://github.com/pulibrary/princeton_ansible/blob/674f6dc4c41fcf91686fa30c03620340d1b1fe9a/roles/php/tasks/main.yml#L19 and I notice curl is already in there.